CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 ### 0.4.4
 * Added a validator for IncidentTypes (incidenttype-*.json).
 * Fixed an issue where the -p flag in the validate was not working.
+* Added the use_cases, global_permissions, command_permissions, additional_info, limitations and troubleshooting arguments to the generate-docs command.
```

Comment:
except permissions, i don't think anyone ever used those arguments, don't know if this needs to be supported

Reply:
Actually I used the limitations and global permissions.
but the troubleshooting is not very useful, I agree.  WDYT @anara123 / @yuvalbenshalom / @Itay4 ?
do we need all new args or can spare some?